### ISSUE TITLE
Parse error

### ISSUE THREAD

**Issue by elyisgreat** [alda-lang/alda#192]
Tried to parse the following in alda, got this error:

`V2: b8. b16 >b</d/f2.`

```
Parse error at line 6, column 16:
V2: b8. b16 >b</d/f2.
               ^
Expected one of:
"*"
#"\z"
"V0:"
"V"
"{"
"["
"|"
"@"
"%"
"⚙"
"<"
">"
"o"
"r"
#"[a-g]"

File load aborted.
```

However, when I try to parse `V2: b8. b16 >b/<d/f2.`, it loads normally.

What's going on? It shouldn't matter on what side of the chord I put the octave advance.

**Comment by daveyarwood** [comment]
This is a bug -- good catch. You're right, you should be able to place octave changes either before or after the slash.

Working on a fix.

**Comment by daveyarwood** [comment]
Fixed in [1.0.0-rc10](https://github.com/alda-lang/alda/releases/tag/1.0.0-rc10). Thanks again @elyisgreat  for reporting this!
